src/Provider/OpenRouteService/composer.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "geocoder-php/open-route-service-provider",
```

Comment:
I would put openrouteservice (without dashes) since it is how they refer to it (see https://github.com/GIScience/openrouteservice)